core/templates/pages/partnerships-page/partnerships-page.component.html

```diff
@@ -1,93 +1,93 @@
-        <img class="oppia-partnerships-process-icon hide-xs" alt="People Icon" [src]="peopleIconUrl">
-      </div>
 
-      <div class="oppia-partnerships-process-subcontainer">
-        <img class="oppia-partnerships-process-icon" alt="Service Icon" [src]="serviceIconUrl">
-        <div class="oppia-partnerships-process-content">
-          <h4 class="oppia-partnerships-h4">{{ 'I18N_PARTNERSHIPS_PAGE_PROCESS_2_HEADING' | translate }}</h4>
-          <p>{{ 'I18N_PARTNERSHIPS_PAGE_PROCESS_2' | translate }}</p>
+        <div class="oppia-partnerships-process-subcontainer">
+          <img class="oppia-partnerships-process-icon" alt="Service Icon" [src]="serviceIconUrl">
+          <div class="oppia-partnerships-process-content">
+            <h4 class="oppia-partnerships-h4">{{ 'I18N_PARTNERSHIPS_PAGE_PROCESS_2_HEADING' | translate }}</h4>
+            <p>{{ 'I18N_PARTNERSHIPS_PAGE_PROCESS_2' | translate }}</p>
+          </div>
         </div>
-      </div>
 
-      <div class="oppia-partnerships-process-subcontainer">
-        <img class="oppia-partnerships-process-icon hide-gt-xs" alt="Agree Icon" [src]="agreeIconUrl">
-        <div class="oppia-partnerships-process-content">
-          <h4 class="oppia-partnerships-h4">{{ 'I18N_PARTNERSHIPS_PAGE_PROCESS_3_HEADING' | translate }}</h4>
-          <p>{{ 'I18N_PARTNERSHIPS_PAGE_PROCESS_3' | translate }}</p>
+        <div class="oppia-partnerships-process-subcontainer">
+          <img class="oppia-partnerships-process-icon hide-gt-xs" alt="Agree Icon" [src]="agreeIconUrl">
+          <div class="oppia-partnerships-process-content">
+            <h4 class="oppia-partnerships-h4">{{ 'I18N_PARTNERSHIPS_PAGE_PROCESS_3_HEADING' | translate }}</h4>
+            <p>{{ 'I18N_PARTNERSHIPS_PAGE_PROCESS_3' | translate }}</p>
+          </div>
+          <img class="oppia-partnerships-process-icon hide-xs" alt="Agree Icon" [src]="agreeIconUrl">
         </div>
-        <img class="oppia-partnerships-process-icon hide-xs" alt="Agree Icon" [src]="agreeIconUrl">
       </div>
     </div>
   </div>
 
   <div class="partering-with-oppia">
-    <h3 class="oppia-partnerships-h3">{{ 'I18N_PARTNERSHIPS_PAGE_PARTNERING_HEADING' | translate }}</h3>
-    <div class="partering-with-oppia-container">
-      <img class="partering-with-oppia-image" alt="Partnerships page image" [src]="partneringImgUrl">
-      <div class="partering-with-oppia-content">
-        <p>{{ 'I18N_PARTNERSHIPS_PAGE_PARTNERING_CONTENT' | translate }}</p>
-        <button class="btn">
-          <a href="https://drive.google.com/file/d/1RZ1mWDA2XWXTh1GlWFf5AWWuV2iGBBa2/view?usp=sharing"
-             target="_blank"
-             rel="noopener">
-             {{ 'I18N_PARTNERSHIPS_PAGE_BROCHURE_BUTTON' | translate }}
-          </a>
-        </button>
+    <div class="content-container">
+      <h3 class="oppia-partnerships-h3">{{ 'I18N_PARTNERSHIPS_PAGE_PARTNERING_HEADING' | translate }}</h3>
+      <div class="partering-with-oppia-container">
```

Comment:
The spelling of "partnering" here is wrong, please fix here and elsewhere.

Reply:
Ah, Sorry my bad
I have made the changes again